http_check/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # CHANGELOG - http_check
 
-## 1.1.0 / Unreleased
+## 1.1.1 / Unreleased
+
+### Changes
+
+* [BUGFIX] Fix response tuple arity in SSL certificate check
```

Comment:
if you add a reference in the footer for #328, make it reference in that line. See other change from `1.1.0`